MerlinAU.sh

```diff
@@ -587,7 +656,7 @@ _SCRIPTUPDATE_()
       if _WaitForYESorNO_
       then
           echo ; echo
-          echo -e "${CYANct}Downloading $SCRIPT_NAME ${CYANct}v$DLRepoVersion${NOct}"
+          echo -e "${CYANct}Downloading $SCRIPT_NAME ${CYANct}v${DLRepoVersion}${NOct}"
```

Comment:
Same with these: $DLRepoVersion$ being detected as a different variable than $DLRepoVersion

Reply:
> Same with these: `$DLRepoVersion$` being detected as a different variable than `$DLRepoVersion`
>
Another good catch!! It is best to follow the best-practice coding standards.